data/virt_autotest/virt-tests.conf

```diff
@@ -0,0 +1,136 @@
+[vmi_lifecycle_test]
+GINKGO_FOCUS=vmi_lifecycle_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vm_test]
+GINKGO_FOCUS=vm_test.go
+GINKGO_SKIP=test_id:3312
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmipreset_test]
+GINKGO_FOCUS=vmipreset_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_configuration_test]
+GINKGO_FOCUS=vmi_configuration_test.go
+SOFT_FAIL=test_id:1681
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_headless_test]
+GINKGO_FOCUS=vmi_headless_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[access_test]
+GINKGO_FOCUS=rfe_id:500
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[storage]
+GINKGO_FOCUS=rfe_id:2298
+GINKGO_SKIP=test_id:3108|test_id:3109
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[container_disk_test]
+GINKGO_FOCUS=container_disk_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[datavolume]
+GINKGO_FOCUS=datavolume.go
+GINKGO_SKIP=test_id:3189|test_id:6686|test_id:5252
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[hotplug]
+GINKGO_FOCUS=hotplug.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_cloudinit_test]
+GINKGO_FOCUS=vmi_cloudinit_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_iothreads_test]
+GINKGO_FOCUS=vmi_iothreads_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_multiqueue_test]
+GINKGO_FOCUS=vmi_multiqueue_test.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[primary_pod_network]
+GINKGO_FOCUS=primary_pod_network.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_multus]
+GINKGO_FOCUS=vmi_multus.go
+EXTRA_OPT=-apply-default-e2e-configuration -skip-shasums-check
+
+[vmi_networking]
+GINKGO_FOCUS=vmi_networking.go
+GINKGO_SKIP=test_id:1780
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[networkpolicy]
+GINKGO_FOCUS=networkpolicy.go
+EXTRA_OPT=-apply-default-e2e-configuration
+
+[vmi_hostdev_test]
```

Comment:
If I remember correctly, this test requires specific host configuration required in test code, which our test machines can not meet.

Reply:
Should I remove its configuration? Or just keep it and enable/disable by a new config option.